### DISCUSSION TITLE
0.7.0 beta go poof?

### DISCUSSION THREAD

**Discussion by RootPC** [bttv-android/bttv#96] [Q&A] [answered]
I keep getting multiple notifications about the 0.7.0 update (about 20 notifs/day) but noticed the 0.7.0 release is empty. Might temporarily disable notifications for the app until released or the placeholder gets deleted.

**Comment by FoseFx** [comment, marked as answer]
Hey! Thank you for your interest in the project.

Version [0.7.0-beta](https://github.com/bttv-android/bttv/releases/tag/v0.7.0-beta) to me looks fine; it exists, it's not in pre-release mode and has an apk file attached.

20 notifications per day is a lot ~~and should not happen~~. ~~Update: I can now confirm this.~~ Update: This is now fixed.

What version of the mod do you have installed? What android version and device are you on? What drew you to the conclusion, that the release is "empty" (e.g. does nothing happen, does the download fail or does the install fail)?

**Reply by RootPC** [reply]
When I tried to open the repository for the 0.7.0 prior to today, it said 'this release is empty' or something along that line. It is now there. I have enabled the 'Allow from this source' to update the app, but it is still not automatically updating. I have an LG v60 running Android 11.

![Screenshot_20210612-122116.png](https://user-images.githubusercontent.com/18724313/121784350-2cead180-cb79-11eb-8895-4ec101ad1756.png)

![Screenshot_20210612-122601.png](https://user-images.githubusercontent.com/18724313/121784389-61f72400-cb79-11eb-81a3-3c5ee3cb7149.png)


Currently running v.0.6.1
